types/index.d.ts

```diff
@@ -1153,6 +1153,19 @@ declare class Realm {
      */
     writeCopyTo(path: string, encryptionKey?: ArrayBuffer | ArrayBufferView): void;
 
+    /**
```

Comment:
Just a question – I notice that the wording between this and the JSDoc is slightly different (because the JSDoc has to cover both cases in one comment). 

If we were to move to generating docs from the Typescript definitions exclusively, are you happy that the docs for both forms of `writeCopyTo` in the Typescript definitions cover everything that is in the JSDoc? It appears so, but just confirming.

Reply:
I believe that it should be covered.  It's just much easier to do in two separate blocks 😄